### ISSUE TITLE
"Security Seal Intact" checkbox is not saved during the sample preservation transition

### ISSUE THREAD

**Issue by leonardorojass** [senaite/senaite.core#1083]
## Steps to reproduce
Tick on the "Security Seal Intact" checkbox before preserving the sample. Then click on "Preserve" or "Save" inside the Sample Partitions section.

## Current behavior
The "Security Seal Intact" checkbox is not saved during the sample preservation transition or when the "Save" button is clicked.

## Expected behavior
When selected the ticked checkbox should be saved.

## Screenshot (optional)
<img width="1261" alt="screen shot 2018-11-11 at 9 17 51 pm" src="https://user-images.githubusercontent.com/2072052/48317840-70af8000-e5f8-11e8-996a-20908f09c700.png">
<img width="1254" alt="screen shot 2018-11-11 at 9 18 01 pm" src="https://user-images.githubusercontent.com/2072052/48317846-74430700-e5f8-11e8-9310-b0bac5486450.png">
<img width="1253" alt="screen shot 2018-11-11 at 9 18 41 pm" src="https://user-images.githubusercontent.com/2072052/48317847-773df780-e5f8-11e8-8418-38366f86aef9.png">

**Comment by ramonski** [comment]
As of 1.3 we removed the partitions. How shall we proceed with that field? Move it to the Sample (AR)?

**Comment by leonardorojass** [comment]
Yes, then it should to be moved to the Sample as it is related to the sample container. 

With the new Sample approach, How does the system handle a sample collected in different containers/preservations and where is this information save/shown in the sample View? In such cases, does a subsample is needed for each container/preservation type? For a sample which contains different AS that requires "Separate Container" (selected in the AS Setup), how does the system handle such cases?
